deploys/reference-app-all-in-one.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: reference-app
+spec:
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 5000
+  selector:
+    run: reference-app
+  sessionAffinity: None
+  type: LoadBalancer
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: reference-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: reference-app
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        run: reference-app
+    spec:
+      containers:
+      - image: gcr.io/genuine-segment-148215/reference-app:v3
```

Comment:
Cool! If this is working: I wonder whether it's possible to load `image: syncforscience/reference-app:v0.1.0` directly (i.e. from the default Docker repository instead of google's) and inject the relevant fixtures file using a `ConfigMap` (and a `volumeMounts` entry on the container template).